code/__defines/damage_organs.dm

```diff
@@ -108,9 +109,11 @@
 #define DROPLIMB_BLUNT 1
 #define DROPLIMB_BURN 2
 
-#define ORGAN_ASSISTED 1 // Like pacemakers, not robotic
-#define ORGAN_ROBOT    2 // Fully robotic, no organic parts
-#define ORGAN_LIFELIKE 3 // Robotic, made to appear organic
+#define MODIFICATION_ORGANIC 0	// Organic
+#define MODIFICATION_ASSISTED 1 // Like pacemakers, not robotic
+#define MODIFICATION_SILICON 2	// Fully robotic, no organic parts
+#define MODIFICATION_LIFELIKE 3	// Robotic, made to appear organic
+#define MODIFICATION_REMOVED 4	// Removed completly
```

Comment:
As is, a `MODIFICATION_LIFELIKE` organ would return TRUE on `BP_IS_ROBOTIC(org)`, but a "robotic" check I see in the code uses `BP_IS_ROBOTIC(org) || BP_IS_LIFELIKE(org)`, for some reason.